src/mutation_events.js

```diff
@@ -143,11 +143,11 @@
   }
 
   function disableMutationEventPolyfill(target) {
-    if (!listeningNodes.has(target))
+    if (hasTarget(listeningNodes, target))
```

Comment:
Should be `!hasTarget(...`

Reply:
Apologies for the mistake, and thanks for catching it.